releasenotes/notes/jmx-fetch-logs-a5ae8bdc27d13a92.yaml

```diff
@@ -0,0 +1,39 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    List upgrade notes here, or remove this section.
```

Comment:
I think you forgot to remove these sections in the reno (maybe the change wasn't staged?)